src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuote.tsx

```diff
@@ -24,49 +23,10 @@ import { useAppDispatch, useAppSelector } from 'state/store'
 
 import { SwapperIcon } from '../SwapperIcon/SwapperIcon'
 
-const TradeQuoteLoading = () => {
```

Comment:
We need to re-add loading state TSX in another PR.

Reply:
i wonder if we should just comment it out for convenience?

Reply:
Or alternatively, `// eslint-disable-next-line @typescript-eslint/no-unused-vars` and a TODO so we don't remove/re-add this component?

Reply:
Missed this - yeah we can do that, will fix up when I'm back at my desk 👌

Reply:
https://github.com/shapeshift/web/pull/4847/commits/5d6b0f26a0da3a763dcb083668eac4c901828be1